dallinger/command_line/docker_ssh.py

```diff
@@ -136,20 +136,6 @@ def prepare_server(host, user):
     else:
         print("Docker daemon already installed")
 
-    try:
```

Comment:
Do we need to check that the installed Docker is a high enough version to support `docker compose`?

Reply:
Tested by sccessfully deploying a PsyNet experiment based on this to be merged branch to a freshly setup server on AWS.

Reply:
Nice, and great to know (via other communication) that you've also tested it on a pre-existing server.